src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -921,4 +920,17 @@ export default class AircraftController {
             this.aircraft_remove(aircraftModel);
         }
     }
+
+    /**
+     * Respond to user changing the tower control (autotower) settings
+     *
+     * @for AircraftController
+     * @private
+     */
+    _onTowerControllerChange() {
+        const isAutoTower = GameController.getGameOption(GAME_OPTION_NAMES.TOWER_CONTROLLER) === 'SYSTEM';
+        for (let i = 0; i < this.aircraft.list.length; i++) {
+            this.aircraft.list[i].onTowerControllerChange(isAutoTower);
```

Comment:
furthermore, we wouldn't even need to pass `isAutoTower` here. the `AircraftModel` itself could handle it all internally

Reply:
... we don't want each aircraft instance making the same redundant `isAutoTower()` call

Reply:
🤦 Holy crap you're right! Ignore this entire line of feedback from me, not sure what I was thinking before. What you did originally is way better.